src/utils/logging.cc

```diff
@@ -54,7 +58,7 @@ void Logging::SetFilename(const std::string& filename) {
   Mutex::Lock lock_(mutex_);
   if (filename_ == filename) return;
   filename_ = filename;
-  if (filename.empty()) {
+  if (filename.empty() || filename == kStderrFilename) {
```

Comment:
if filename is kStderrFilename - the buffer should be pumped to std::cerr.

Reply:
Indeed!